scripts/docker/run-local.sh

```diff
@@ -1,3 +1,9 @@
 #! /usr/bin/env sh
 
-docker run -i -t --privileged=true --rm -v gd-fs:/gd-fs:ro gooddata-fs
+if [ $# -ge 3 ]; then
+    host_mountpoint=$3
+else
+    host_mountpoint=gd-fs
```

Comment:
The command gooddata-fs will fail then anyway, so this is just a dummy to let the error pass through from gooddata-fs and not duplicate it here.